ffi/kotlin_experimental/pom.xml

```
@@ -0,0 +1,158 @@
+        <kotlin.version>2.1.0</kotlin.version>
+        <junit.jupiter.version>5.9.2</junit.jupiter.version>
+        <junit.platform.version>1.14.0</junit.platform.version>
+    </properties>
+    <dependencies>
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+            <version>3.8.1</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.jetbrains.kotlin</groupId>
+            <artifactId>kotlin-test-junit5</artifactId>
+            <version>1.7.0</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.jetbrains.kotlin</groupId>
+            <artifactId>kotlin-stdlib</artifactId>
+            <version>${kotlin.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>net.java.dev.jna</groupId>
+            <artifactId>jna</artifactId>
+            <version>5.14.0</version>
+        </dependency>
+        <dependency>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter-api</artifactId>
+            <version>${junit.jupiter.version}</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.junit.platform</groupId>
+            <artifactId>junit-platform-engine</artifactId>
+            <version>${junit.platform.version}</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter-engine</artifactId>
+            <version>${junit.jupiter.version}</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.junit.platform</groupId>
+            <artifactId>junit-platform-launcher</artifactId>
+            <version>${junit.platform.version}</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>io.mockk</groupId>
+            <artifactId>mockk-jvm</artifactId>
+            <version>1.13.10</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>net.jqwik</groupId>
+            <artifactId>jqwik-kotlin</artifactId>
+            <version>1.9.3</version>
+            <scope>test</scope>
+        </dependency>
+
+    </dependencies>
+    <build>
+        <sourceDirectory>${project.basedir}/src/main/kotlin</sourceDirectory>
+        <resources>
+          <resource>
+            <!-- Ideally this can be taught to run cargo build. Use basedir for now so that it does the same thing
+                 as the gradle build. -->
+            <filtering>false</filtering>
+            <directory>${basedir}/src/main/native</directory>
+            <includes>
+              <include>libicu4x.so</include>
+              <include>libicu4x.dylib</include>
+              <include>libicu4x.dll</include>
```

```suggestion
              <include>icu4x.dll</include>
```

The resulting `.dll` is `icu4x.dll` not `libicu4x.dll` because the `Makefile` says:
```
ifeq ($(OS),Windows_NT)
  DYLIB_EXT = "dll"
  DYLIB_PREFIX = ""
else
...
endif

...

cp ../../target/release/$(DYLIB_PREFIX)icu_capi.$(DYLIB_EXT) src/main/native/$(DYLIB_PREFIX)icu4x.$(DYLIB_EXT)
```